### ISSUE TITLE
Update navigation items under "Projects" in the global navigation

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#7258]
## Description

Make updates to the global navigation under 'Projects'.

Reference the original bug [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1522698)
Use [this mock-up](https://docs.google.com/presentation/d/1jn9DKaZjE07jr0PWiABXhxMFs2Awl3TPLOpn7qURvpM/edit#slide=id.p) as the final version for updates. It contains a link to the new thumbnail and URLs for each project, along with updated copy

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Tagged for L10n

#### Tasks
- [ ] Update copy and URLs
- [ ] Update thumbnail with[ this image](https://drive.google.com/file/d/1jaummnRKpOZyNzuq_S9U3D9_rFwo5uAK/view?usp=sharing)

**Comment by hoosteeno** [comment]
@slightlyoffbeat, I know this issue is fairly ancient, but would you say it's still a worthwhile change to make, or should we look for fresher guidance about this nav item?

**Comment by slightlyoffbeat** [comment]
@hoosteeno this still needs updating, but it'd be good to take a fresh look at this. I can help with this.

**Comment by hoosteeno** [comment]
@slightlyoffbeat Can you update this with fresh specs?

**Comment by slightlyoffbeat** [comment]
@hoosteeno 

Updated specs on latest comment here:

https://bugzilla.mozilla.org/show_bug.cgi?id=1522698

One note: I provided two options for the featured block. I'd like to go with option 1, but I'd also like to have option two translated and ready to go in the event that we need to swap things out quickly.

**Comment by alexgibson** [comment]
Adding a blocked label to this based on activating Fluent in Pontoon.

**Comment by alexgibson** [comment]
This is now unblocked - @slightlyoffbeat @hoosteeno are the changes here still up-to-date / current?

**Comment by ejregithub** [comment]
@hoosteeno - can we move on this work?

**Comment by hoosteeno** [comment]
Let's go with @slightlyoffbeat 's specs from https://bugzilla.mozilla.org/show_bug.cgi?id=1522698#c6 -- that is, two string variations get into localization pipes.

**Comment by alexgibson** [comment]
@hoosteeno @slightlyoffbeat here's the new content on demo - please verify if it all looks correct? https://www-demo3.allizom.org/en-US/

I went with option 1 (Mozilla Labs) for the thumbnail link. Let me know if this is ok, or you would like to try the other option? I'm going to suggest we pick one option to localize, as not being able to see content being translated can sometimes be confusing for those doing the L10n work.

**Comment by hoosteeno** [comment]
I think this looks good, and I might suggest one more change while we are modifying this part of the nav.

In https://github.com/mozilla/bedrock/issues/9231, the requester says labs.mozilla.org is unmaintained, and suggests redirecting traffic to https://mozilla.dev/.

@slightlyoffbeat do you think we should replace the labs CTA in the Projects dropdown with a link to mozilla.dev?

@alexgibson do you think we should bundle that work here for any reason, or file a new issue?

**Comment by alexgibson** [comment]
> @alexgibson do you think we should bundle that work here for any reason, or file a new issue?

Happy to bundle here if it makes sense to do so

**Comment by alexgibson** [comment]
> In https://github.com/mozilla/bedrock/issues/9231, the requester says labs.mozilla.org is unmaintained, and suggests redirecting traffic to https://mozilla.dev/.

@hoosteeno thinking about this further, https://mozilla.dev/ might make more sense under Developers? With that in mind, It might be better to handle labs being obsolete in a separate issue, as it might imply more changes that are beyond the original scope here.

Perhaps we should consider "Mozilla Mixed Reality" (option 2) for the thumbnail under projects, as suggested in https://bugzilla.mozilla.org/show_bug.cgi?id=1522698#c6

Let me know what you think.

**Comment by alexgibson** [comment]
@hoosteeno I updated the thumbnail to "Mozilla Mixed Reality": https://www-demo3.allizom.org/en-US/

Please let me know if this is good to move forward with, thanks

**Comment by hoosteeno** [comment]
It sounds good. Thanks.
